src/server/v2.0/handler/auditlog.go

```diff
@@ -39,18 +41,23 @@ func (a *auditlogAPI) ListAuditLogs(ctx context.Context, params auditlog.ListAud
 	}
 
 	if !secCtx.IsSysAdmin() {
-		// ToDo remove the dependency on GetMyProjects()
-		projects, err := secCtx.GetMyProjects()
-		if err != nil {
-			return a.SendError(ctx, fmt.Errorf(
-				"failed to get projects of user %s: %v", secCtx.GetUsername(), err))
-		}
 		ol := &q.OrList{}
-		for _, project := range projects {
-			if a.HasProjectPermission(ctx, project.ProjectID, rbac.ActionList, rbac.ResourceLog) {
-				ol.Values = append(ol.Values, project.ProjectID)
+		if sc, ok := secCtx.(*local.SecurityContext); ok {
```

Comment:
Is this intentional? Previously it doesn't require the securityContext must be local based.